### PR TITLE
Notes on how to install and use git

### PR DESCRIPTION
**Pull request by kenny-cfg**
This is a fabulous PR, please approve. Don't merge yet.

### CODE REVIEW

**Comment by claritalyo** [comment]
👀

File: cheat.sheets/git.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# git
+
+## Installation
+
+Instructions on how to install `choco` or `brew` can be found [here](./install.software.md)
```
Comment (sandramolina) [inline cheat.sheets/git.md]: Can you please explain what you are doing here?

**Review by sandramolina — CHANGES_REQUESTED** [review]

File: cheat.sheets/git.md (hunk 1 above)
Comment (kenny-cfg) [reply cheat.sheets/git.md]: I've added some explanation here.

**Review by sandramolina — APPROVED** [review]
LGTM

**Pull request merged**